gitrevise/todo.py

```diff
@@ -154,17 +154,19 @@ def edit_todos_msgedit(repo: Repository, todos: List[Step]) -> List[Step]:
          f, fixup <commit> = use commit, but fuse changes into previous commit
          s, squash <commit> = like fixup, but also edit the commit message
          c, cut <commit> = interactively split commit into two smaller commits
-         i, index <commit> = leave commit changes unstaged
+         i, index <commit> = stage the change, i.e. add to the index
 
         Each command is prefixed by a '++' marker, and followed by the complete
         commit message.
 
         Commit messages will be reworded to match the text following them
         before the command is performed.
 
-        If a command is removed, it will be treated like an 'index' line.
+        These blocks can be re-ordered; they are executed from top to bottom. If
+        present, then index blocks must be at the bottom of the list, i.e. they
+        can not be followed by non-index blocks. Also, their messages are lost.
```

Comment:
In the rest of the comment I refer to them as "commands", so let's keep using that wording in this comment.